### PR TITLE
Update Cypress tests

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description
- Updated UI tests
- Re organized UI tests

## Type of change

Please delete options that are not relevant.

- [ ] New tests (non-breaking change which adds functionality)

# How Has This Been Tested?

- Tested against the deployed version of the app in dev environment

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

Test Results: 

malt-test-results

### CODE REVIEW

File: maltd-frontend/cypress/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Run tests from maltd-frontend directory.
 ### Run E2E test on electron headless mode
 
 ```
-npm run cy:run -- --spec "cypress/integration/tests/end-to-end-flow.js"
+npm run cy:run -- --spec "cypress/integration/tests/end-to-end-functional-tets.js"
```
Comment (sdevalapurkar) [inline maltd-frontend/cypress/README.md]:
```suggestion
npm run cy:run -- --spec "cypress/integration/tests/end-to-end-functional-test.js"
```

Comment (sdevalapurkar) [reply maltd-frontend/cypress/README.md]: I think you may need to rename a file

File: maltd-frontend/cypress/integration/tests/end-to-end-functional-tests.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/* global cy, Cypress */
+// / <reference types="Cypress" />
+
+describe("MALT Ui Functional Test Suite", () => {
```
Comment (sdevalapurkar) [inline maltd-frontend/cypress/integration/tests/end-to-end-functional-tests.js:4]:
```suggestion
describe("MALTD UI Functional Test Suite", () => {
```

**Review by sdevalapurkar — APPROVED** [review]
awesome work

**Review by pbolduc — APPROVED** [review]

**Pull request merged**